controllers/imagejob/imagejob_controller.go

```diff
@@ -144,6 +144,7 @@ func checkNodeFitness(pod *corev1.Pod, node *corev1.Node) bool {
 
 //+kubebuilder:rbac:groups=eraser.sh,resources=imagejobs,verbs=get;list;watch;create;update;patch;delete
 //+kubebuilder:rbac:groups=eraser.sh,resources=imagejobs/status,verbs=get;update;patch
+//+kubebuilder:rbac:groups="",resources=configmaps,verbs=create;get;list;watch
```

Comment:
Should we add a namespace here? we don't want it hardcoded since charts can be deployed on any namespace though

Reply:
is there a way to pass a string into the kubebuilder comment? Im not sure if there is a way without hardcoding

Reply:
I was thinking it could be hardcoded here but we can override in the helm chart. is that possible?

Reply:
I tested it by hardcoding the namespace but I don't think it will be possible since eraser-manager-role doesn't apply to eraser-system which is the role we are using to access